### PR TITLE
Harigovind/ch103/rapid value increment decrement

### PR DESCRIPTION
**Pull request by harigovind1998**
Increments/decrements every half a second for the first 2 or 3 seconds then starts increasing/decreasing event tenth of a second. The time it takes to go into rapid mode varies a little due to the nature of handler/runnables. I had to restrict the toast messages when going above 100 or negative values to just once because if someone held the decrement/increment button down after the max/min was reached, it kept showing the toast.

### CODE REVIEW

File: app/src/main/java/ca/chronofit/chrono/util/listeners/RepeatListener.kt [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    private var touchView: View? = null
+    @Volatile
+    private var counter = 0
+
+    private val runnable = object : Runnable {
+        override fun run() {
+            if (touchView!!.isEnabled) {
+                if(counter > 3){
+                    handler.postDelayed(this, FAST_INCREMENT_PERIOD)
+                }else{
```
Comment (SVD6) [inline app/src/main/java/ca/chronofit/chrono/util/listeners/RepeatListener.kt]: Auto format please

**Review by SVD6 — CHANGES_REQUESTED** [review]
I haven't taken a full look at the code just yet, looks mostly fine from what I can tell. Autoformat it real quick and push a commit with that. 

After using the feature itself, I feel like the initial increment is a bit too slow. Either that or the time it takes to switch from slow to fast should be less. Play around with it with a certain use case in mind. For example, say you want to have 100 seconds, use that to test it out and tweak it till it makes the most sense. The goal is to make it as easy as possible for someone to for example get it to 100 seconds with just the plus button.

When it gets to it's max, it keeps going on. I don't know if it's resetting to 0 and incrementing or if it's going past 999. It's weird, take a look at it please. I went to 900 and held down the plus sign (if you want to recreate it).

I'll take a closer look at the code itself after you make these changes^

File: app/src/main/java/ca/chronofit/chrono/util/listeners/RepeatListener.kt (hunk 1 above)
Comment (harigovind1998) [reply app/src/main/java/ca/chronofit/chrono/util/listeners/RepeatListener.kt]: Addressed.
Decreased the time it takes to go from "slow rapid increment" to "fast rapid increment". LMK how you like this. I feel like if i decrease time between increments was any faster, the user will easily overshoot the time they want.

**Review by SVD6 — APPROVED** [review]
Ship it

**Pull request merged**